jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenterTest.java

```diff
@@ -140,4 +146,35 @@ public void testSetupDetailBreadcrumb() {
                                           eq(Commands.DO_NOTHING));
     }
 
+    @Test
+    public void bulkReleaseOnlyOnReservedOrInProgressTasksTest() {
+        List<TaskSummary> taskSummaries = new ArrayList<>();
+        taskSummaries.add(createTestTaskSummary(TASK_ID, TASK_STATUS_RESERVED, identity.getIdentifier()));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 1, TASK_STATUS_RESERVED, "otherUser"));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 2, TASK_STATUS_READY, ""));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 3, TASK_STATUS_IN_PROGRESS, ""));
+
+        getPresenter().bulkRelease(taskSummaries);
+
+        verify(taskService, times(3)).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), anyLong());
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID));
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID + 1));
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID + 3));
+    }
+
+    @Test
+    public void bulkSuspendOnlyOnReserveInProgressReadyTest() {
```

Comment:
The method name indicates that it would also test tasks in states that should not be suspended, but here all the tasks can be. Does it make sense to add another task in different state?

Reply:
@jstastny-cz sure, added a new completed task to test